playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx

```diff
@@ -42,7 +45,7 @@ const TextInput = (props: TextInputProps) => {
     placeholder,
     required,
     type = 'text',
-    value,
+    value = '',
```

Comment:
Defaulting this fixes the warning that mentions invalid use of Controlled/Uncontrolled inputs. What this further means is that now any example devoid of state management will simply not update when typed into, so I am also updating the docs by adding that state management where need be.